### PR TITLE
Rbac tests

### PR DESCRIPTION
**Pull request by ray-milkey**

### CODE REVIEW

**Comment by ray-milkey** [comment]
retest this please

File: test/rbac/badtokentest.go [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+		{
+			name:                "Bad Password",
+			username:            "alicec",
+			password:            "not-the-password",
+			token:               "",
+			expectedClientError: "401",
+		},
+		{
+			name:                "Bad User",
+			username:            "go-ask-alice",
```
Comment (SeanCondon) [inline test/rbac/badtokentest.go:54]: :grinning:

File: test/rbac/notokentest.go [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
...iOiJSUzI1NiIsInR5cCIgOiAiSldUIiwia2lkIiA6ICJ5bWkyYUZqS2RGRTdkY1VobnV4QzhHU2Y2NXM5SVpjUm1zVWtMVDBmaHZBIn0.eyJleHAiOjE2NDE5MzMxMDIsImlhdCI6MTY0MTkzMzA0MiwianRpIjoiMzkxMzkzZTctM2FhZC00ZmQzLThlNTUtOGI5OTQ1ZmVhMTMyIiwiaXNzIjoiaHR0cHM6Ly9rZXljbG9hay1kZXYub25sYWIudXMvYXV0aC9yZWFsbXMvbWFzdGVyIiwic3ViIjoiNDAzOTdmZWQtNzc2Ny00ZmM4LTg3YzEtOWI4MTY2MTU1NDdhIiwidHlwIjoiQmVhcmVyIiwiYXpwIjoib25vcy1jb25maWctdGVzdCIsInNlc3Npb25fc3RhdGUiOiJjODViNWE4MC0zY2IwLTQ1MTgtOWI2YS00NDQ4YWU4NGU0MzciLCJhY3IiOiIxIiwic2NvcGUiOiJvcGVuaWQgZW1haWwgcHJvZmlsZSBncm91cHMiLCJzaWQiOiJjODViNWE4MC0zY2IwLTQ1MTgtOWI2YS00NDQ4YWU4NGU0MzciLCJlbWFpbF92ZXJpZmllZCI6dHJ1ZSwibmFtZSI6IkFsaWNlIEFkbWluIiwiZ3JvdXBzIjpbIkFldGhlclJPQ0FkbWluIiwibWl4ZWRHcm91cCJdLCJwcmVmZXJyZWRfdXNlcm5hbWUiOiJhbGljZWEiLCJnaXZlbl9uYW1lIjoiQWxpY2UiLCJmYW1pbHlfbmFtZSI6IkFkbWluIiwiZW1haWwiOiJhbGljZWFAb3Blbm5ldHdvcmtpbmcub3JnIn0.eaVrlbX9XXOQg006_N4Y_OJ6MXBMUPepPRd6PZGWbhieEafRv4CvlCL6Jy2W4tdTmkY5bjXLLXwQdum3iesJKJ8QcXlgoTc1N1MwyfZAChMLtuN1y75VAfWKLL5fctpP8ZlFC3xDj84IM46KpYUiwocv1TGFrTE0LNd2xC5bB3ajmiZ69KyetJjBZ04xvLfKcY88XEXNVb-FqoRaoCiMavqio9Dk_2CBVzbdVvv-mtvGfYJzPE3dEadkKG8w8P2o46Iv8FL_NjaNPNm0_LZAtOPx3KMRYZOBnR0-Gd2p1eyN-h4VZr9T2k0u5dm-7bmxTWNTwyF_PP7bUIUJhS6gYw"
```
Comment (SeanCondon) [inline test/rbac/notokentest.go:32]: you're not using this

**Review by SeanCondon — APPROVED** [review]

**Comment by ray-milkey** [comment]
retest this please

**Pull request merged**